tracer/src/Datadog.Trace/ContinuousProfiler/ProfilerStatus.cs

```diff
@@ -63,6 +65,12 @@ private void EnsureNativeIsIntialized()
 
                 _isInitialized = true;
 
+                if (!ProfilerAvailabilityHelper.IsContinuousProfilerAvailable)
+                {
+                    Log.Debug("The continuous profiler is not available in this environment.");
```

Comment:
Don't we want a log all the time because it could help quickly investigating possible issues without asking for debug logs?

Reply:
Yeah, I was torn, but it probably makes sense 👍

Reply:
```suggestion
                    Log.Information("The continuous profiler is not available in this environment.");
```